lib/driver.js

```diff
@@ -17,6 +17,7 @@ class WindowsDriver extends BaseDriver {
     this.jwpProxyAvoid = NO_PROXY;
 
     this.opts.address = opts.address || DEFAULT_WAD_HOST;
+    this.opts.createSessionTimeout = opts.createSessionTimeout || DEFAULT_CREATE_SESSION_TIMEOUT_MS;
```

Comment:
what's the point to import and assign the default value here if we can do the same inside winappdriver module?